utils/dataloaders.py

```diff
@@ -52,53 +55,69 @@ def get_fashion_mnist_dataloaders(batch_size=128,
     return train_loader, test_loader
 
 
-def get_dsprites_dataloader(batch_size=128,
-                            path_to_data=os.path.join(DIR, '../data/datadsprites/dsprites_data.npz')):
+def get_dsprites_dataloader(batch_size=128):
```

Comment:
i think it's a lot cleaner to keep a `path_to_data` argument instead of forcing the directory structure you want. but I guess it's fine